scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/helpers/StartAtUpdater.java

```diff
@@ -56,30 +73,39 @@ public boolean updateStartAt(InternalJob job, String startAt, SchedulerDBManager
 
     }
 
-    private Set<TaskId> updateStartAtAndTasksScheduledTime(InternalJob job, String startAt, long scheduledTime) {
+    private Set<TaskId> updateStartAtAndTasksScheduledTime(InternalJob job, String startAt, long scheduledTime,
+            ExtendedSchedulerPolicy schedulerPolicy) {
 
         List<InternalTask> internalTasks = job.getITasks();
         Set<TaskId> updatedTasks = new HashSet<>(internalTasks.size());
 
-        if (resetJobGenericInformation(job, startAt)) {
+        if (resetJobGenericInformation(job, startAt, schedulerPolicy)) {
 
             for (InternalTask td : internalTasks) {
                 td.setScheduledTime(scheduledTime);
                 updatedTasks.add(td.getId());
                 job.getJobDescriptor().updateTaskScheduledTime(td.getId(), scheduledTime);
+                if (schedulerPolicy != null) {
+                    schedulerPolicy.updateStartAt(td.getId().toString(), startAt);
+                }
             }
         }
 
         return updatedTasks;
     }
 
-    private boolean resetJobGenericInformation(InternalJob job, String startAt) {
+    private boolean resetJobGenericInformation(InternalJob job, String startAt,
```

Comment:
I don't see a case when it will be false

Reply:
ok, only when isValidStartAt(genericInformation, startAt) returns false the fucntion will return false

Reply:
I don't understand to which code you refer to ...

Reply:
@mboussaa can I merge my PR as it is ?